src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -236,24 +237,19 @@ protected function addFluentIndexes()
     /**
      * Add the fluent commands specified on any columns.
      *
+     * @param  \Illuminate\Database\Connection  $connection
      * @param  \Illuminate\Database\Schema\Grammars\Grammar  $grammar
      * @return void
      */
-    public function addFluentCommands(Grammar $grammar)
+    public function addFluentCommands(Connection $connection, Grammar $grammar)
     {
         foreach ($this->columns as $column) {
-            foreach ($grammar->getFluentCommands() as $commandName) {
-                $attributeName = lcfirst($commandName);
-
-                if (! isset($column->{$attributeName})) {
```

Comment:
Why is this check removed?

Reply:
Before this before, we had only one fluent command which was comment on PostgreSQL, this PR does a clean up on auto-increment starting values and add that as fluent command for PostgreSQL and MySQL, also adds default as a fluent command for SQL Server. This Check is removed because it won't work for all these commands, instead of checking the value here, we check the situation on the command itself and decide whether to compile or not.

fluent commands before this PR: Comment (PostgreSQL)
fluent commands after this PR: Comment (PostgreSQL), Auto-increment starting values (PostgreSQL and MySQL) and Default (SQL Server)